lib/transports/__es__/_script.js

```diff
@@ -0,0 +1,81 @@
+const jsonParser = require('../../jsonparser.js')
+const aws4signer = require('../../aws4signer')
+const async = require('async')
+const _ = require('lodash')
+
+class Script {
+  getScripts (limit, offset, callback) {
+    if (this.gotScripts === true) {
+      callback(null, [])
+    } else {
+      const esRequest = {
+        url: `${this.base.host}/${this.base.index}/_scripts/${this.base.type || '*'}`,
```

Comment:
the type has to be specified, so there is no '*' default since ES doesn't support the retrieval of all stored scripts. https://github.com/elastic/elasticsearch/issues/88947

Reply:
ok, maybe a list of scripts to be dumped can be passed as parameters.